templates/docs/patterns/pagination.md

```diff
@@ -44,8 +52,21 @@ To import just this component into your project, copy the snippet below and incl
 @include vf-p-icon-contextual-menu;
 ```
 
-For more information see [Customising Vanilla](/docs/customising-vanilla/) in your projects, which includes overrides and importing instructions.
+### Providing context
```

Comment:
We do refer to this pattern as `article pagination` in the docs, so would be good if heading uses that, so it shows up in side navigation.